bin/github-deploy.sh

```diff
@@ -151,10 +151,6 @@ if [ "$(echo "${PROCEED:-n}" | tr "[:upper:]" "[:lower:]")" != "y" ]; then
   exit 1
 fi
 
-# Version changes
-output 2 "Updating version numbers in files and regenerating php autoload classmap (note pre-releases will not have the readme.txt stable tag updated)..."
```

Comment:
What about keeping this step too? It could be an extra check (someone could forget to run the new command).